lib/setupmanagers/hckclient.rb

```diff
@@ -201,10 +197,9 @@ def reset_to_ready_state
       set_machine_ready
     end
 
-    def abort
+    def close
       @logger.info("Aborting HLKClient #{@name}")
```

Comment:
"Closing" instead of "Aborting" ?

Reply:
Done with commit 0ff4aa59654e0ba1b04be37fa23276b6a43f03e9, but I used word "exiting" instead; it's a bit awkward to say closing something not a file. I just named the method "close" to match the interface; there is no such restriction for a log message.